landoapi/commit_message.py

```diff
@@ -198,3 +198,11 @@ def split_title_and_summary(msg: str) -> Tuple[str, str]:
     tail = parts[1:]
     summary = "\n".join(tail).strip()
     return title, summary
+
+
+def bug_list_to_commit_string(bug_ids: List[str]) -> str:
+    """Convert a list of `str` bug IDs to a string for a commit message."""
+    if not bug_ids:
+        return "No bug"
+
+    return f"Bug {', '.join(bug_ids)}"
```

Comment:
Probably worth supporting singular and plural forms here. E.g.

```
Bug XXXX
Bug XXXX, Bug YYYY, Bug ZZZZ
```

Do we have an existing convention for multiple bugs? Thinking of things that parse commit messages that look for this form.